conformance/system_macros/annotate.ion

```diff
@@ -0,0 +1,9 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Test Cases:
+// annotate can be invoked by name, address, qualified address, qualified name, binary address, and binary system e-expression op code.
```

Comment:
Should this be factored into its own test and applied to all system macros?

Reply:
I think it's easier to include it here because we can't easily parameterize a single test with all the system macros because they all have different signatures.